Added the full-sync procedure (#26)
desmos-labs/docs: docs/03-fullnode/02-setup.md

* If you're running a *full node*:
    1. Run `desmos unsafe-reset-all`;
    2. Restart the node.

## 6. Full sync - Sync from block 1
:::warning Full sync from scratch

> With version `2.3.1` it's no longer possible to sync the node from scratch. If, for any reason, you need to do it,
> please follow one of these procedure below:

Reviewer: ```suggestion
With version `2.3.1` it's no longer possible to sync the node from scratch. If, for any reason, you need to do it,
please follow the procedure below.
```

:::

:::caution   
Remember that this procedure takes time (hours, or days) and you will not be able to perform any operation on the node in the meantime.
:::

### 1. Downgrade the software
The first thing you need to do in order to start sync from scratch is getting the correct Desmos version according to the network you're connecting to:
* [**Mainnet**](../06-mainnet/full-sync.md);
* [**Testnet**](../05-testnets/03-join-public/full-sync.md).

### 2. Disable state-sync (if needed)

> * Open the `~/.desmos/config/config.toml` file;
> * Disable state sync by setting `enable = false`;
> * Run a `desmos unsafe-reset-all`.

Reviewer: ```suggestion
1. Open the `~/.desmos/config/config.toml` file.
2. Disable state sync by setting `enable = false`.
3 Run `desmos unsafe-reset-all`.
```

> ### 3. Setup Cosmovisor (if needed)

Reviewer: ```suggestion
### 3. Setup Cosmovisor
```

> Since we're relying on the cosmos-SDK Upgrade module to update the network, you will need to setup cosmovisor

Reviewer: ```suggestion
Since we're relying on the Cosmos SDK `x/upgrade` module to update the network, you will need to setup Cosmovisor
```
